apps/e2e/cypress/e2e/proposals.cy.ts

```diff
@@ -1616,4 +1619,154 @@ context('Proposal tests', () => {
+      // This prevents flaky tests with some long-running network requests from one test to finish in the next and unexpectedly update the app.
+      cy.window().then((win) => {
+        win.location.href = 'about:blank';
+      });
+      cy.resetDB();
+      cy.getAndStoreFeaturesEnabled();
+      cy.createTemplate({
+        name: 'Proposal Template with Instrument Picker',
+        groupId: TemplateGroupId.PROPOSAL,
+      }).then((result) => {
+        if (result.createTemplate) {
+          createdTemplateId = result.createTemplate.templateId;
+
+          cy.createTopic({
+            templateId: createdTemplateId,
+            sortOrder: 1,
+          }).then((topicResult) => {
+            if (!topicResult.createTopic) {
+              throw new Error('Can not create topic');
+            }
+
+            topicId =
+              topicResult.createTopic.steps[
+                topicResult.createTopic.steps.length - 1
+              ].topic.id;
+            cy.createQuestion({
+              categoryId: TemplateCategoryId.PROPOSAL_QUESTIONARY,
+              dataType: DataType.INSTRUMENT_PICKER,
+            }).then((result) => {
+              instrumentPickerQuestionId = result.createQuestion.id;
+              cy.updateQuestion({
+                id: instrumentPickerQuestionId,
+                question: instrumentPickerQuestion,
+                config: `{"variant":"dropdown","isMultipleSelect":false,"required":true,"requestTime":true}`,
+              });
+              cy.createQuestionTemplateRelation({
+                questionId: instrumentPickerQuestionId,
+                templateId: createdTemplateId,
+                sortOrder: 0,
+                topicId: topicId,
+              });
+            });
+          });
+        }
+      });
+      cy.createCall({
+        ...newCall,
+        allocationTimeUnit: AllocationTimeUnits.DAY,
+        proposalWorkflowId: initialDBData.workflows.defaultWorkflow.id,
+        templateId: createdTemplateId,
+      }).then((response) => {
+        if (response.createCall) {
+          createdCallId = response.createCall.id;
+          cy.createInstrument(instrument).then((result) => {
+            cy.assignInstrumentToCall({
+              callId: createdCallId,
+              instrumentFapIds: [{ instrumentId: result.createInstrument.id }],
+            });
+          });
+          cy.createInstrument(instrument2).then((result) => {
+            cy.assignInstrumentToCall({
+              callId: createdCallId,
+              instrumentFapIds: [{ instrumentId: result.createInstrument.id }],
+            });
+          });
+        }
+      });
+      cy.login('officer');
+      cy.visit('/');
+    });
+    it('Should be able to download proposal pdf for a proposal which contains instrument picker question', function () {
+      if (!featureFlags.getEnabledFeatures().get(FeatureId.SCHEDULER)) {
```

Comment:
is there any reason we are not running these in e2e mode and only stfc mode?

Reply:
The test case was failing without this check and now I don't remember the message shown for the failed test case. 
However when I checked in other places like [here](https://github.com/UserOfficeProject/user-office-core/blob/4ab6e3c9984d1ba180dbfce7e84b3ac20b35b504/apps/e2e/cypress/e2e/proposalAdministration.cy.ts#L474) and [here](https://github.com/UserOfficeProject/user-office-core/blob/4ab6e3c9984d1ba180dbfce7e84b3ac20b35b504/apps/e2e/cypress/e2e/proposalAdministration.cy.ts#L505) I found this check was there, so I added that in the new test case and it worked.

Reply:
This flag is often used to check which config we are running in (e2e vs stfc) so we are skipping the test in e2e mode (which aligns more closly with ess config) since this is not something that is stfc specific (correct me if im wrong) I think we should have it working in both modes